src/docs/product/accounts/quotas/spike-protection.mdx

```diff
@@ -60,7 +60,11 @@ We recommend taking the following steps to manage your spikes:
 - Set up an [on-demand budget](/product/accounts/quotas/manage-event-stream-guide/#on-demand-budget) to make sure you have time to adjust your volume in the event of a future spike.
 - Set up [spend allocations](/product/accounts/quotas/#spend-allocation) to ensure your critical projects are guaranteed a portion of your reserved volume, even if there are spikes in other projects.
 
-To review the events that were dropped to save your quota with Spike Protection, go to the "Usage Stats" tab of the **Stats** page for your Sentry org and select the desired event in the "Category" dropdown. You can see project-level details of your stats by using the project selector. Sentry can show the spike threshold for a time duration greater than 6hrs and less than 30 days.
+To review the events that were dropped to save your quota with Spike Protection, go to the "Usage Stats" tab of the **Stats** page for your Sentry org and select the event of interest from the "Category" dropdown. You'll be able to see project-level details of your stats by using the project selector.
+
+To ensure accuracy, only spikes that are over an hour long and fall into a time range that's between 6 hours and 30 days will be viewable from the **Stats** page. You'll be able to gage any spikes against a spike threshold in the form of a dotted line as long as the time range is between 6 hours and 30 days.
```

Comment:
can we mention the time period on the stats page specifically? the time range mentioned here almost sounds like the spike length or the spike threshold length

do we want to mention ongoing spikes?